test/models/movie.test.js

```diff
@@ -1,9 +1,84 @@
 'use strict';
 
+const Knex  = require('../../lib/libraries/knex');
 const Movie = require('../../lib/models/movie');
 
+const oldMovie  = { release_year: 1950, name: 'Lob Interns 1950' };
+const newMovie  = { release_year: 2017, name: 'Lob Interns 2017' };
+const bestMovie = { release_year: 2016, name: 'Lob Interns 2016' };
```

Comment:
Nit: `s/bestMovie/secondBetMovie`.

I also think you're missing the following:
```js
const bestMovie = { release_year: 2015, name: 'Lob Interns 2015' };
```